examples/app-basic/package.json

```
@@ -5,8 +5,7 @@
   "scripts": {
     "build": "webpack -c ./.config/webpack/webpack.config.ts --env production",
     "dev": "webpack -w -c ./.config/webpack/webpack.config.ts --env development",
-    "e2e:update": "npm exec cypress install && npm exec grafana-e2e run --update-screenshots",
-    "e2e": "npm exec cypress install && npm exec grafana-e2e run",
+    "e2e": "npx playwright test",
```

I don't think we need to use npx here. Additionally something we need to consider is if anyone uses this as a "starter" for building an app create-plugin updates (as of now) will replace/remove this script.

```suggestion
    "e2e": "playwright test",
```